### ISSUE TITLE
[sign in with apple]does this package require minimum iOS version?

### ISSUE THREAD

**Issue by SeasonLeee** [aboutyou/dart_packages#152]
I believe there shouldn't be a minimum version of iOS to use sign in with apple...

However, when I install the example project on a iOS device with version 12.4.8, the console of XCode just output error.

I'm going to paste the error message below, hope you guys can help me to tackle this, thanks!

```
2020-09-28 11:22:04.182697+0800 Runner[569:354073] Metal API Validation Enabled
2020-09-28 11:22:04.967524+0800 Runner[569:354344] flutter: Observatory listening on http://127.0.0.1:53230/fOQfI7eQopY=/
2020-09-28 11:22:12.958246+0800 Runner[569:354335] [VERBOSE-2:ui_dart_state.cc(166)] Unhandled Exception: SignInWithAppleNotSupportedException(Unsupported platform version: iOS 12.4.8)
#0      SignInWithApple.getAppleIDCredential (package:sign_in_with_apple/src/sign_in_with_apple.dart:137:7)
<asynchronous suspension>
#1      _MyAppState.build.<anonymous closure> (package:sign_in_with_apple_example/main.dart:30:58)
#2      GestureRecognizer.invokeCallback (package:flutter/src/gestures/recognizer.dart:184:24)
#3      TapGestureRecognizer.handleTapUp (package:flutter/src/gestures/tap.dart:524:11)
#4      BaseTapGestureRecognizer._checkUp (package:flutter/src/gestures/tap.dart:284:5)
#5      BaseTapGestureRecognizer.handlePrimaryPointer (package:flutter/src/gestures/tap.dart:219:7)
#6      PrimaryPointerGestureRecognizer.handleEvent (package:flutter/src/gestures/recognizer.dart:477:9)
#7      PointerRouter._dispatch (package:flutter/src/gestures/pointer_router.dart:78:12)
#8      PointerRouter._dispatchEventToRoutes.<anonymous closure> (package:flutter/src/gestures/pointer_router.dart:124:9)
#9      _LinkedHashMapMixin.forEach (dart:collection-patch/compact_hash.dart:377:8)
#10     PointerRouter._dispatchEventToRoutes (package:flutter/src/gestures/pointer_router.dart:122:18)
#11     PointerRouter.route (package:flutter/src/gestures/pointer_router.dart:108:7)
#12     GestureBinding.handleEvent (package:flutter/src/gestures/binding.dart:220:19)
#13     GestureBinding.dispatchEvent (package:flutter/src/gestures/binding.dart:200:22)
#14     GestureBinding._handlePointerEvent (package:flutter/src/gestures/binding.dart:158:7)
#15     GestureBinding._flushPointerEventQueue (package:flutter/src/gestures/binding.dart:104:7)
#16     GestureBinding._handlePointerDataPacket (package:flutter/src/gestures/binding.dart:88:7)
#17     _rootRunUnary (dart:async/zone.dart:1206:13)
#18     _CustomZone.runUnary (dart:async/zone.dart:1100:19)
#19     _CustomZone.runUnaryGuarded (dart:async/zone.dart:1005:7)
#20     _invoke1 (dart:ui/hooks.dart:267:10)
#21     _dispatchPointerDataPacket (dart:ui/hooks.dart:176:5)
2020-09-28 11:24:00.002916+0800 Runner[569:354073] Could not load IOSurface for time string. Rendering locally instead.
(lldb) 
```

**Comment by SeasonLeee** [comment]
Hi there, I hope I didn't cause any panic there.

So, after I have done my research, I realized that this 'sign in with apple' thing is kind of have iOS version difference. 

Those built up on iOS 13 is based on that  AuthenticationServices, and the other iOS is based on something else...

Either way, I have talked our product manager and we had the same agreement that if the iOS is and lower than 12, there'll be no button. 

To sum up, thank you guys for this amazing feature!

**Comment by HenriBeck** [comment]
The native Sign in with Apple flow is only available on iOS 13 or newer operating systems. There was some work done by an external contributor to have the same web flow-based authentication for iOS 12 and below, but we decided that it's not worth the long term support to accept this feature.

The adoption rate of iOS is excellent, so the shares of iOS 12 will just keep continuing to decrease with iOS 14 being out now.

I am closing the ticket then.
